packages/compass-query-bar/src/components/query-bar.spec.tsx

```diff
@@ -295,6 +296,11 @@ describe('QueryBar Component', function () {
     it('renders the query history popover', function () {
       const queryHistory = screen.getByTestId(queryHistoryComponentTestId);
       expect(queryHistory).to.be.visible;
+
+      // Note: We re-close the query history as it attempts to focus trap after
+      // react-testing-library has unmounted the content which causes an error.
+      const button = screen.getByTestId(queryHistoryButtonId);
+      button.click();
```

Comment:
Does this mean that if we quickly mount and unmount component in the application (like user opening and quickly closing the popover) it will also throw? Should we figure out why this is happening instead of working around it?

Reply:
It's happening because of the focus trap not testing nicely without full dom, we could disable display checks, which is what we previously had.
 https://github.com/focus-trap/focus-trap-react#testing-in-jsdom 
 Slightly related to this thread https://github.com/mongodb-js/compass/pull/3571#discussion_r992611971
 I'll investigate a bit more, maybe passing `fallbackFocus` is what we want end up going to (or back to disabling display check). Will ping when changed.

Reply:
Fixed with fallbackFocus 👍